### ISSUE TITLE
Ember-Mocha + Ember-Cli-Mocha = invalid /tests HTML

### ISSUE THREAD

**Issue by mike-north** [emberjs/ember-mocha#211]
A project with both `ember-cli-mocha` and `ember-mocha` as top-level ember addons will result in duplicate fixture HTML being rendered. Unfortunately, this also means that more than one DOM element has the same `id`. [This is not allowed.](https://www.w3.org/TR/html5/dom.html#element-attrdef-global-id)

![screen shot 2018-07-31 at 11 20 06 am](https://user-images.githubusercontent.com/558005/43479107-74c59d90-94b4-11e8-930b-4a629b1883d7.png)

The end result is that tests will pass, but the developer will not be able to easily inspect the rendered app in the 50% scaled frame (the second set of DOM elements) due to the real testing being done in the first set of DOM elements.

#### Possible root cause

the contentFor hook of this addon is
https://github.com/emberjs/ember-mocha/blob/13be23a36a6af0f6f1cffe934b905a2bbe1fd39e/index.js#L66-L77

and for `ember-cli-mocha`
https://github.com/ember-cli/ember-cli-mocha/blob/64d8742b7fdc8535f76b545fb7fc7c6d6c97b01c/index.js#L10-L13
```ts
  contentFor(type) {
    let output = this.eachAddonInvoke('contentFor', [type]);
    return output.join('\n');
  },
```
It's possible that this results in `ember-mocha`'s fixture HTML being inserted twice

**Comment by Turbo87** [comment]
`ember-cli-mocha` and `ember-mocha` should not be installed at the same time. `ember-cli-mocha` already brings its own `ember-mocha` dependency, but ultimately we should just use `ember-mocha` directly from now on

**Comment by mike-north** [comment]
@Turbo87 any reason why we shouldn't check for invalid dependency situations in this addon's `included` hook?

**Comment by Turbo87** [comment]
no, makes sense to check it. though we need to be careful to not warn for the `app -> ember-cli-mocha -> ember-mocha` dependency scenario.
